.graphqlconfig.yaml

```diff
@@ -0,0 +1,42 @@
+projects:
+  users:
```

Comment:
I guess you are unhappy with the naming too. It is confuses me to call it users where it exposes more. Or what was your reasoning? Internal would also be a strange name. Maybe mcInternal or mcApi.

Reply:
Yeah, couldn’t find a better name. We can also use `mc`

Reply:
I couldn’t really stop thinking about it. I would likely also suggest to go with mc. Just to align it with the GraphQL targets we have. So it is easy to map and correlate even if the name is not perfect.